README.md

```diff
@@ -231,6 +237,51 @@ By default, `beekeeper-scheduler-apiary` listens on port 8080, `beekeeper-path-c
 | `scheduler-delay-ms`                | No       | Amount of time (in milliseconds) between consecutive cleanups. Default value is `300000` (5 minutes after the previous cleanup completes). |
 | `Metastore-uri`                     | Yes      | URI of the Hive Metastore where tables to be cleaned-up are located. |
 
+## Beekeeper-API
+
+Beekeeper also has an API which provides access to the Beekeeper database and allows to see what metadata and paths are currently being managed.
+
+This allows the user to manually enter a database and a table name and check whether this table has been successfully registered in Beekeeper along with the TTL configuration: the current status of the table, the date and time it will be deleted, the current cleanup delay... etc.
+
+It currently supports two endpoints; one for the expired metadata and another one for the unreferenced paths.
+
+It also supports different filters (see [filtering section](#filtering)).
+
+### Expired metadata endpoint (`GET /metadata`)
+
+This endpoint will return the TTL configuration of all expired partitions that are going to be deleted (or have been deleted) in a specific table. If it is unpartitioned it will just show one object; the table.
+
+It is available in this url; 
+
+    http://beekeeper-api.<address>/api/v1/database/{databaseName}/table/{tableName}/metadata
+
+ where `{databaseName}` and `{tableName}` must be replaced by the database and table name the user wants to search for. So for example, if they wanted to check a table called `my_cool_table` in the database `my_cool_database`, they would go to 
+ 
+    http://beekeeper-api.<address>/api/v1/database/my_cool_database/table/my_cool_table/metadata
```

Comment:
probably best to mention the base url once above `http://beekeeper-api.<address>/api/v1/` and then in these sections specify the rest of the url, like `<base>/api/v1/database/my_cool_database/table/my_cool_table/metadata` that way if the base needs updating in the future (e.g. v2) it can be done in one place